content/en/docs/tutorials/security/cluster-level-pss.md

```diff
@@ -0,0 +1,319 @@
+---
+title: Apply Pod Security Standards at the Cluster Level
+content_type: tutorial
+weight: 10
+---
+
+{{% alert title="Note" %}}
+This tutorial applies only for new clusters.
+{{% /alert %}}
+
+Pod Security admission (PSA) is enabled by default in v1.23 and later, as it [graduated
+to beta](/blog/2021/12/15/pod-security-admission-beta/). Pod Security Admission
+is an admission controller that applies Pod Security Standards when pods are 
+created. This tutorial shows you how to enforce the `baseline` Pod Security 
+Standard at the cluster level which applies a standard configuration 
+to all namespaces in a cluster.
+
+For applying pod security standards one namespace at a time, please [follow this
+tutorial](/docs/tutorials/security/ns-level-pss).
+
+## {{% heading "prerequisites" %}}
+
+Install the following on your workstation:
+
+- [KinD](https://kind.sigs.k8s.io/docs/user/quick-start/#installation)
+- [kubectl](https://kubernetes.io/docs/tasks/tools/)
+
+## Choose the right Pod Security Standard to apply
+
+[Pod Security Admission](/docs/concepts/security/pod-security-admission/)
+lets you apply built-in [Pod Security Standards](/docs/concepts/security/pod-security-standards/)
+with the following modes: `enforce`, `audit`, and `warn`.
+
+To gather information that helps you to choose the Pod Security Standards
+that are most appropriate for your configuration, do the following:
+
+1. Create a cluster with no Pod Security Standards applied:
+
+    ```shell
+    kind create cluster --name psa-wo-cluster-pss --image kindest/node:latest
+    ```
+    The output is similar to this:
+    ```
+    Creating cluster "psa-wo-cluster-pss" ...
+    ✓ Ensuring node image (kindest/node:latest) 🖼
+    ✓ Preparing nodes 📦  
+    ✓ Writing configuration 📜
+    ✓ Starting control-plane 🕹️
+    ✓ Installing CNI 🔌
+    ✓ Installing StorageClass 💾
+    Set kubectl context to "kind-psa-wo-cluster-pss"
+    You can now use your cluster with:
+    
+    kubectl cluster-info --context kind-psa-wo-cluster-pss
+    
+    Thanks for using kind! 😊
+    
+    ```
+
+2. Set the kubectl context to the new cluster:
```

Comment:
nit, but specifying `1.` for every number in an ordered list (instead of 1 2 3 4 etc) in Markdown reduces future maintenance overhead, since list items can be added/removed without having to adjust the rest of the items in the list.